spec/controllers/user/registration_controller_spec.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe User::RegistrationsController, type: :controller do
+  before do
+    @request.env["devise.mapping"] = Devise.mappings[:user]
+
+    allow(APP_CONFIG).to receive(:dig).with(:hcaptcha, :enabled).and_return(true)
+    allow(controller).to receive(:verify_hcaptcha).and_return(captcha_successful)
+  end
+
+  describe "#create" do
+    context "valid user sign up" do
```

Comment:
You can create a second `before` block inside the `valid user sign up` context block where you do the method stubbing.